src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise.component.ts

```diff
@@ -1,56 +1,70 @@
+/* angular */
```

Comment:
We don't use this kind of comment anywhere atm.
We could add linting rules if we want to enforce a specific order.

Reply:
Yes, you are right. I deleted it